tutorials/tutorial_views.py

```diff
@@ -0,0 +1,944 @@
+    elif platform == "darwin":
+        pag.hotkey('option', 'command', 'm')
+        print("\n INFO : Automation is running on Mac OS..\n")
+    elif platform == "win32":
+        pag.hotkey('win', 'd')
+        print("\n INFO : Automation is running on Windows OS..\n")
+    else:
+        pag.alert(text="Sorry, no support on this platform!", title="Platform Exception", button='OK')
+    print("\n INFO : We will be learning how to create a project in MSS with all the views.\n")
+
+
+def call_recorder():
+    """
+    Calls the screen recorder class to start the recording of the automation.
+    """
+    sr.main()
+
+
+def call_mss():
+    """
+    Calls the main MSS GUI window since operations are to be performed on it only.
+    """
+    mss_pyui.main()
+
+
+def automate_views():
+    """
+    This is the main automating script of the MSS views tutorial which will cover all the views(topview, sideview,
+    tableview, linear view) in demonstrating how to create a project. This will be recorded and savedto a file having
+     dateframe nomenclature with a .mp4 extension(codec).
+    """
+    # Giving time for loading of the MSS GUI.
+    pag.sleep(5)
+
+    if platform == 'linux' or platform == 'linux2' or platform == 'darwin':
+        dir_path = 'pictures/tutorial_views/win/'
+        wms_path = 'pictures/tutorial_wms/linux/'
+    elif platform == 'win32':
+        dir_path = 'pictures/tutorial_views/win/'
+        wms_path = 'pictures/tutorial_wms/win/'
+
+    # Screen Resolutions
+    sc_width, sc_height = pag.size()[0] - 1, pag.size()[1] - 1
+
+    # Maximizing the window
+    try:
+        if platform == 'linux' or platform == 'linux2':
+            pag.hotkey('winleft', 'up')
+        elif platform == 'darwin':
+            pag.hotkey('ctrl', 'command', 'f')
+        elif platform == 'win32':
+            pag.hotkey('win', 'up')
+    except Exception:
+        print("\nException : Enable Shortcuts for your system or try again!")
+    pag.sleep(2)
+    pag.hotkey('ctrl', 'h')
+    pag.sleep(2)
+
+    # Shfting topview window to upper right corner
+    try:
+        x, y = pag.locateCenterOnScreen(f'{dir_path}add_waypoint.png')
+        pag.click(x, y - 56, interval=2)
+        if platform == 'win32' or platform == 'darwin':
+            pag.dragRel(525, -110, duration=2)
+        elif platform == 'linux' or platform == 'linux2':
+            pag.dragRel(910, -25, duration=2)
+        pag.move(None, 56)
+        add_tv_x, add_tv_y = pag.position()
+        pag.move(-486, -56, duration=1)
+        pag.click(interval=1)
+        if platform == 'win32' or platform == 'linux' or platform == 'linux2':
+            pag.hotkey('ctrl', 'v')
+        elif platform == 'darwin':
+            pag.hotkey('command', 'v')
+        pag.sleep(4)
+        # Shifting Sideview window to upper left corner.
+        try:
+            x1, y1 = pag.locateCenterOnScreen(f'{dir_path}add_waypoint.png')
+            if platform == 'win32' or platform == 'darwin':
```

Comment:
the pag code lines can be moved to a utils functions and has the platform as input value

Reply:
Okay. I will try to find a way in which it can be done. Since, there are many small things which are platform dependent and even if when added to utils function, somethings still need to be there in the source code regarding platform.

Reply:
you can do this later, when you have a module ready. I guess it is better readable afterwards.